modules/_labs/pagination/react/lib/GoTo.tsx

```diff
@@ -0,0 +1,70 @@
+import styled from '@emotion/styled';
+import {type} from '@workday/canvas-kit-react-core';
+import TextInput from '@workday/canvas-kit-react-text-input';
+import React from 'react';
+
+const GoToLabel = styled('div')({
+  margin: 'auto',
+  paddingRight: '8px',
+  ...type.body,
+  ...type.variant.label,
+});
+
+const GoToWrapper = styled('div')({
+  display: 'flex',
+  paddingLeft: '1px',
+});
+
+const InputWrapper = styled('div')({
+  minWidth: '10px',
+  maxWidth: '45px',
+});
+
+const GoTo: React.FC<{
+  onSubmit: (page: number, e: React.SyntheticEvent) => void;
```

Comment:
I think the property interface should be an interface like the other components.

Also, I thought it was odd to have the event as the second parameter. Other components that have handlers list the event as the first parameter. Also we prefer `event` to `e`. This is part of the external API of the `GoTo` component.

Reply:
I went ahead and made the props an interface. I actually don't really see a need for passing event so I removed it. If you'd like me to add it back, I can.